blog/_posts/2021-11-29-tilt-news-november-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: "tilt-news-november-2021"
+date: 2021-11-29
+author: lian
+layout: blog
+title: "Tilt News, November 2021"
+image: "/assets/images/tilt-news-november-2021/pexels-frans-van-heerden-624015.jpg"
+image_caption: "Aurora Borealis behind an igloo. Photo by Frans Van Heerden from <a href='https://www.pexels.com/photo/aurora-borealis-624015/'>Pexels</a>"
+description: "introducing unsung heroes and another How-To video! 🎉"
+youtubeId: Wur9HPjZ2Jo
+tags:
+  - news
+---
+Bonjour Tilteurs et Tilteuses,
+
+December is just around the corner, which means it’s time for the end of the year madness again. Hard to believe 2021 is almost over, isn’t it? We’re planning a cool 2021 recap for the end of the year, so there’ll be no December newsletter 😢  
```

Comment:
```suggestion
December is just around the corner, which means it’s time for the end of the year madness again. Hard to believe 2021 is almost over, isn’t it? We’re planning a cool 2021 recap for the end of the year, so we'll have that instead of a December newsletter!  
```

Please don't make me sad right in the beginning :)